NEWS.md

```diff
@@ -1,5 +1,7 @@
 # usethis (development version)
 
+`use_rscloud_badge()` is a new function that creates a README badge indicating the repository can be launched in an [RStudio Cloud](https://rstudio.cloud) project (@gvelasq, #1589).
```

Comment:
We usually prefer to list the issue number. Or the issue number and the PR number.

Reply:
Thank you for clarifying. Fixed in b30dc78bfc4d86c034057ec4f192eff25e5fe947.